src/main/java/pages/Homepage.java

```diff
@@ -0,0 +1,64 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import pages.Loginpage;
+
+
+import static utils.WaitUtils.waitUntilElementContainsText;
+import static utils.WaitUtils.waitUntilElementsIsPresented;
+
+public class Homepage {
+    private final By createAccountHeaderLink = By.cssSelector(".header.links:list-child");
+
+    private final By signInHeaderLink = By.cssSelector(".header.links .authorization-link");
+    private final By welcomeText = By.className("logged-in");
+
+    private final By mainBannerTopText = By.cssSelector("span[class='info']");
+    private final By mainBannerMainText = By.cssSelector("strong[class='title']");
+    private final By mainBannerButtonText = By.className(".action.more.button");
```

Comment:
There are invalid selectors for: above 3 elements. The correct ones are:
private final By mainBannerTopText = By.cssSelector(".info");
private final By mainBannerMainText = By.cssSelector(".title");
private final By mainBannerButtonText = By.cssSelector(".action.more.button");

If you're using By.className you should pass class name not css selector.